integration-tests/conftest.py

```diff
@@ -1,10 +1,27 @@
 import pytest
+import os
+import subprocess
 
 
+@pytest.fixture(scope="session", autouse=True)
+def install_katello_rpm(test_config):
+    if "satellite" in os.getenv('ENV_FOR_DYNACONF'):
+        # install katello rpm before register system against Satellite
+        satellite_hostname = test_config.get("candlepin", "host")
+        cmd = 'rpm -qa | grep katello | xargs rpm -e;rpm -Uvh http://%s/pub/katello-ca-consumer-latest.noarch.rpm' %satellite_hostname
+        ret, _ = subprocess.getstatusoutput(cmd)
```

Comment:
`getstatusoutput()` is a very old API, I'd get away from it :)

A better replacement is `subprocess.call()`, or even better `subprocess.check_call()` since we need both the commands to succeed. Please note that, in case you run shell commands, you need to pass `shell=True`.